src/test/java/com/blackducksoftware/integration/hub/alert/AlertConstantsTest.java

```diff
@@ -0,0 +1,33 @@
+package com.blackducksoftware.integration.hub.alert;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+import org.junit.Test;
+
+import javassist.Modifier;
+
+public class AlertConstantsTest {
+
+    @Test
+    public void testInstantiationException() throws NoSuchMethodException, SecurityException {
```

Comment:
Why are we testing reflective constructors? Is this one of those weird edge cases where the class has no defined methods, but still claims line coverage is incomplete?

Reply:
This seems odd because we hide the default constructor in AlertConstants because it should only contain static fields.  We throw the exception as a precaution.  That's what he's trying to test here.